src/components/common/TransactionRow.js

```diff
@@ -133,154 +135,124 @@ export class TransactionRowComponent extends React.PureComponent<Props> {
-                </T>
-                <T style={isSentTransaction ? styles.transactionDetailsSentTx : styles.transactionDetailsReceivedTx}>{cryptoAmountString}</T>
+        <TouchableOpacity onPress={this.handlePress}>
+          <View style={styles.rowContainer}>
+            <View style={styles.iconContainer}>{thumbnailPath ? <Image style={styles.icon} source={{ uri: thumbnailPath }} /> : transactionIcon}</View>
+            <View style={styles.transactionContainer}>
+              <View style={styles.transactionRow}>
+                <EdgeText style={styles.transactionText}>{transactionText}</EdgeText>
+                <EdgeText style={isSentTransaction ? styles.negativeCryptoAmount : styles.positiveCryptoAmount}>{cryptoAmountString}</EdgeText>
               </View>
-              {formattedTransactionCategory ? (
-                <View style={styles.transactionDetailsRow}>
-                  <T style={styles.transactionCategory}>{formattedTransactionCategory}</T>
-                  <T style={styles.transactionFiat}>{fiatAmountString}</T>
-                </View>
-              ) : null}
-              {formattedTransactionCategory ? (
-                <View style={[styles.transactionDetailsRow, styles.transactionDetailsRowMargin]}>
-                  <T style={[styles.transactionPendingTime, pendingStyle]}>{pendingText}</T>
-                </View>
-              ) : null}
-              {!formattedTransactionCategory ? (
-                <View style={styles.transactionDetailsRow}>
-                  <T style={[styles.transactionPendingTime, pendingStyle]}>{pendingText}</T>
-                  <T style={styles.transactionFiat}>{fiatAmountString}</T>
+              <View style={styles.transactionRow}>
+                <View style={styles.categoryAndTimeContainer}>
+                  {categoryText && <EdgeText style={styles.category}>{categoryText}</EdgeText>}
+                  <EdgeText style={pendingStyle}>{pendingText}</EdgeText>
                 </View>
-              ) : null}
+                <EdgeText style={styles.fiatAmount}>{fiatAmountString}</EdgeText>
+              </View>
             </View>
           </View>
-        </TouchableHighlight>
+        </TouchableOpacity>
       </View>
     )
   }
 }
 
-const rawStyles = {
-  singleTransaction: {
-    height: scale(80),
-    borderBottomWidth: 1,
-    borderBottomColor: THEME.COLORS.GRAY_3,
-    padding: scale(15),
-    paddingRight: scale(15),
-    paddingLeft: scale(15)
-  },
-  singleTransactionWrap: {
-    backgroundColor: THEME.COLORS.WHITE,
-    flexDirection: 'column',
+const getStyles = cacheStyles((theme: Theme) => ({
+  container: {
     flex: 1
   },
-  singleDateArea: {
-    backgroundColor: THEME.COLORS.GRAY_4,
-    flex: 3,
-    padding: scale(3),
-    paddingLeft: scale(15),
+  rowContainer: {
     flexDirection: 'row',
-    paddingRight: scale(24)
+    justifyContent: 'center',
+    paddingVertical: theme.rem(0.25),
+    paddingHorizontal: theme.rem(6 / 16),
```

Comment:
We should not be using REM sizes More granular than 0.25. can you see how this looks at 0.25

Reply:
This is fixed on the commit of 2.0 design